lib/puppet/http/client.rb

```diff
@@ -140,14 +151,19 @@ def close
 
   private
 
-  def execute_streaming(request, user: nil, password: nil, ssl_context:, include_system_store:, &block)
+  def execute_streaming(request, options, &block)
```

Comment:
when I put `options: {}` here to make `options` optional, I got a bunch of failures. Trying to diagnose that.